src/types.jl

```diff
@@ -101,6 +101,16 @@ struct ProjectedGradientAscent{T}
     stepsize::T
 end
 
+"""
+    ProjectedVariance{S,T}
+    Project variance such that for whatever method used,
+    minimum variance does not go below varfloor
```

Comment:
```suggestion
Project variance such that for whatever method used,
minimum variance does not go below varfloor
```
Only indent the first line.